core/src/models/Property.ts

```diff
@@ -361,6 +361,25 @@ export class Property extends LoggedModel<Property> {
     );
   }
 
+  @AfterSave
+  static async ensureRecordsMarkedPendingOnTypeChange(instance: Property) {
+    // Check to see if type changed
+    if (instance.changed("type") && instance.state === "ready") {
```

Comment:
Does somewhere else do this same thing on changing the options or filter?

For better or worse, they seem to do 
```
await PropertyOps.enqueueRuns(this);
```
Should that be done in this case, too?
Should the "pending" work have been done in those cases, too?